airbyte-config/persistence/src/main/java/io/airbyte/config/persistence/ConfigWriter.java

```diff
@@ -0,0 +1,155 @@
+/*
+ * Copyright (c) 2021 Airbyte, Inc., all rights reserved.
+ */
+
+package io.airbyte.config.persistence;
+
+import io.airbyte.commons.enums.Enums;
+import io.airbyte.commons.json.Jsons;
+import io.airbyte.config.StandardDestinationDefinition;
+import io.airbyte.config.StandardSourceDefinition;
+import io.airbyte.db.instance.configs.jooq.Tables;
+import io.airbyte.db.instance.configs.jooq.enums.ActorType;
+import io.airbyte.db.instance.configs.jooq.enums.ReleaseStage;
+import io.airbyte.db.instance.configs.jooq.enums.SourceType;
+import java.time.LocalDate;
+import java.time.OffsetDateTime;
+import java.util.List;
+import org.jooq.DSLContext;
+import org.jooq.JSONB;
+import org.jooq.impl.DSL;
+
+public class ConfigWriter {
```

Comment:
I'd probably add some top-level javadoc to this class explaining the motivation/recommended usage of it. From this PR, I can tell that this class is being used as a sort of intermediary between the DatabaseConfigPersistence class (which we're moving away from) and the ConfigRepository class (which we're moving towards), but as an outside reader it probably isn't clear where I should be looking for/adding new persistence logic.

To be clear, I like the approach of moving private methods out of the DatabaseConfigPersistance class, I just think this specific class could use some javadoc so other developers can quickly understand where it fits into the picture

Reply:
makes sense, will add